pkg/controller/node/ipam/cidrset/cidr_set_test.go

```diff
@@ -130,6 +130,69 @@ func TestIndexToCIDRBlock(t *testing.T) {
 			CIDRBlock:      "2001:db8:85a3:3232::/63",
 			description:    "Index with IPv6",
 		},
+		{
+			clusterCIDRStr: "2001:0db8::/32",
+			subnetMaskSize: 48,
+			index:          0,
+			CIDRBlock:      "2001:db8::/48",
+			description:    "Index with IPv6",
```

Comment:
With the increased number of tests, can the description be a bit different so make it easier to identify a failing test?

Reply:
Done